build.sh

```diff
@@ -275,7 +275,7 @@ function install_caffe2() {
       CMAKE_ARGS+=("-DCUDA_ARCH_NAME='Maxwell'")
     fi
 
-    if [[ $(which ccache | wc -c) -ne 0 ]]; then
+    if [[ $(which ccache | wc -c) -ne 0 && -e ${CCACHE_WRAPPER_DIR}/nvcc ]]; then
```

Comment:
I was going to approve this as obvious, but after a closer look, the original check
should simply be replaced.  The fact that there is some `ccache` is completely irrelevant.

Maybe something like
`-L ${CCACHE_WRAPPER_DIR}/nvcc && $($(readlink ${CCACHE_WRAPPER_DIR}/nvcc) --version | grep ccache | wc -c) -ne 0`

Btw, I fixed a minor typo in the other commit message.

Reply:
I don't have a ccache with nvcc so I canot test if that would work...